include/rocksdb/write_buffer_manager.h

```diff
@@ -34,7 +34,7 @@ class StallInterface {
   virtual void Signal() = 0;
 };
 
-class WriteBufferManager {
+class WriteBufferManager final {
```

Comment:
I like this! One of my previous PR had this assumption in mind that WBM should not be extended and made a API implement change in [here](https://github.com/facebook/rocksdb/pull/8506/files#diff-5531a9905804e3a40070a32c4f36e3952d09b23c6ca42099074fc180b02d7954R77). It's good to explicitly mark it final just to confirm my assumption!